src/createNumbersGenerator.js

```diff
@@ -25,8 +25,18 @@
  *
  * @return {number}
  */
-function createNumbersGenerator(numbers, min, max) {
-  // write code here
+function createNumbersGenerator(numbers = [], min = -Infinity, max = Infinity) {
+  const filteredNum = numbers.filter((item) => item >= min && item <= max);
```

Comment:
better to check numbers before using the method
```suggestion
  const filteredNum = numbers ? numbers.filter((item) => item >= min && item <= max) : undefined;
```